.github/workflows/GHA-Scala-Instrumentation-Tests.yaml

```diff
@@ -130,25 +108,33 @@ jobs:
           rm gradle.properties
           mv gradle.properties.gha gradle.properties
 
-      - name: Run instrumentation tests for Java ${{ matrix.java-version }} and scala ${{ matrix.scala }}
-        if: ${{ env.GRADLE_KEY != '' }}
+      - name: Run instrumentation tests for Java ${{ matrix.java-version }} and scala ${{ matrix.scala }} (attempt 1)
+        id: run_tests_1
+        continue-on-error: true
         env:
-          CI: GHA
           JAVA_HOME: ${{ env.ORG_GRADLE_PROJECT_jdk8 }}
-          GRADLE_ENTERPRISE_ACCESS_KEY: 34.229.76.228=${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}
-          GRADLE_CACHE_USER: gha
-          GRADLE_CACHE_PASSWORD: ${{ secrets.GRADLE_CACHE_PASSWORD }}
-          JRE: jre${{ matrix.java-version }}
-          LANGUAGE: scala
-          SCALA_VERSION: scala${{ matrix.scala }}
-          TEST_TYPE: instrumentation
-        run: ./gradlew --console=plain --parallel :instrumentation:test -Ptest${{ matrix.java-version }} -PnoInstrumentation -Pscala-${{ matrix.scala }} --continue --scan -Dscan.uploadInBackground=false --build-cache
-
-      - name: Run instrumentation tests for Java ${{ matrix.java-version }} and scala ${{ matrix.scala }} w/o GE
-        if: ${{ env.GRADLE_KEY == '' }}
+        run: ./gradlew --console=plain --parallel :instrumentation:test -Ptest${{ matrix.java-version }} -PnoInstrumentation -Pscala-${{ matrix.scala }} --continue
+        timeout-minutes: 25
+
+      - name: Run instrumentation tests for Java ${{ matrix.java-version }} and scala ${{ matrix.scala }} (attempt 2)
+        id: run_tests_2
+        if: steps.run_tests_1.outcome == 'failure'
+        continue-on-error: true
         env:
           JAVA_HOME: ${{ env.ORG_GRADLE_PROJECT_jdk8 }}
-        run: ./gradlew --console=plain --parallel :instrumentation:test -Ptest${{ matrix.java-version }} -PnoInstrumentation -Pscala-${{ matrix.scala }} --continue
+        run: |
+          sleep 60
```

Comment:
why do you need this sleep?

Reply:
I put the sleep in to give a pause for any potential transient problems that might have caused the failure.  Maybe unnecessary?

Reply:
I didn't do that in the Java Inst. test, but I saw it used on an example of retries that gave that reason. I'm not tied to it if you don't think it's necessary.

Reply:
I would do without it. If we see that many retries are failing, then we can introduce.